services/src/unit-test/java/uk/gov/moj/sdt/services/cache/ErrorMessagesCacheTest.java

```diff
@@ -30,13 +30,16 @@
  * $LastChangedBy: $ */
 package uk.gov.moj.sdt.services.cache;
 
-import static org.junit.Assert.fail;
-
-import org.easymock.EasyMock;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-
+import static org.junit.jupiter.api.Assertions.*;

```

Comment:
Generally good practice to explicitly state the imports that are required rather than using a wildcard import. Replace this with the explicit imports that are needed.

Reply:
organized imports

Reply:
Confirmed imports are now all explicit.